content/docs/for-developers/sending-email/laravel.md

```diff
@@ -94,7 +94,7 @@ In Laravel `Views` are used as 'templates' when sending an email. Let's create a
     	</head>
     	<body>
     		<h2>Test Email</h2>
-    		<p>{{ $message }}</p>
+    		<p>{{ $testmessage }}</p>
```

Comment:
Thanks so much for the update. Before we merge this, would you mind updating `$testmessage` to `$test_message` to match coding style? Thanks again for your work on this.

Reply:
I renamed the variable. Please take a look. Thanks.